C4G.Unity/Assets/C4G/Editor/Scripts/C4GSettings.cs

```diff
@@ -38,7 +45,44 @@ public string SerializedConfigsFolderFullPath
 
         public bool IsGeneratedCodeFolderValid => !string.IsNullOrEmpty(_generatedCodeFolderPath) &&
                                                   Directory.Exists(GeneratedCodeFolderFullPath);
+
         public bool IsSerializedConfigsFolderValid => !string.IsNullOrEmpty(_generatedCodeFolderPath) &&
-                                                  Directory.Exists(SerializedConfigsFolderFullPath);
+                                                      Directory.Exists(SerializedConfigsFolderFullPath);
+
+        public IReadOnlyList<string> SheetNames => _sheetNames;
+
+        private readonly List<IAliasDefinition> _aliasDefinitionsConvertBuffer = new List<IAliasDefinition>();
+        public IReadOnlyList<IAliasDefinition> AliasDefinitions
+        {
+            get
+            {
+                _aliasDefinitionsConvertBuffer.Clear();
+                foreach (SerializedAliasDefinition serializedAliasDefinition in _aliasDefinitions)
+                {
+                    string parserTypeFullName = serializedAliasDefinition.Parser.ParserTypeFullName;
+                    if (!string.IsNullOrEmpty(parserTypeFullName) && C4GTypeParserSerializationHelper.TypeIndexByAssemblyQualifiedNameMap.TryGetValue(parserTypeFullName, out int parserTypeIndex))
+                    {
+                        Type parserType = C4GTypeParserSerializationHelper.ParserTypes[parserTypeIndex];
+                        IC4GTypeParser parser = (IC4GTypeParser)Activator.CreateInstance(parserType);
+                        AliasDefinition aliasDefinition = new AliasDefinition(serializedAliasDefinition.Name, parser);
+                        _aliasDefinitionsConvertBuffer.Add(aliasDefinition);
+                    }
+                }
+
+                return _aliasDefinitionsConvertBuffer;
+            }
+        }
+
+        private readonly struct AliasDefinition : IAliasDefinition
+        {
+            public AliasDefinition(string name, IC4GTypeParser parser)
+            {
+                Name = name;
+                Parser = parser;
+            }
+
+            public string Name { get; }
```

Comment:
[Critical] I would suggest to review all structs in the project and replace members with `readonly field` instead of property. Easy save on useless vertial method calling :)